index.html

```diff
@@ -25,7 +25,7 @@
   <body>
     <div>
       <!-- Good default -->
-      <protvista-uniprot accession="P11166"></protvista-uniprot>
+      <protvista-uniprot accession="P05067"></protvista-uniprot>
```

Comment:
Did you mean to make this change?

Reply:
Yes, P11166 was tested from previous change and published with it. I reverted back to our familiar entry